sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -253,6 +253,13 @@ class RocksDB(
     logInfo(s"Rolled back to $loadedVersion")
   }
 
+  def cleanup(): Unit = {
```

Comment:
When will we call this method?

Reply:
It will be called in the `RocksDBStateStoreProvider.doMaintenace`. I'll submit the state store provider PR (the last one) today.

Reply:
okay.